### ISSUE TITLE
Unity Asset

### ISSUE THREAD

**Issue by eusebiu** [Hotrian/OVRdrop-Public#73]
Is there any way to integrate OVRdrop in an existing Unity scene (VR app)?

**Comment by Hotrian** [comment]
No, at this time OVRdrop is intended for be a standalone application.

**Comment by syrsly** [comment]
He could use the open source old version and modify it as needed.

**Comment by Hotrian** [comment]
Currently OVRDrop and OVRDDP are not licensed for use in other projects, so that would be okay for personal use, but not for something released publicly.

**Comment by Hotrian** [comment]
Currently OVRdrop and OVRDDP are not licensed for use in other projects OVRDDP should be used as a learning tool and not as a base or plugin for other applications.

**Comment by Hotrian** [comment]
Currently OVRdrop and OVRDDP are not licensed for use in other projects. OVRDDP should be used as a learning tool and not as a base or plugin for other applications. OVRDDP is using some other code such as the [HSVPicker](https://github.com/judah4/HSV-Color-Picker-Unity) which are licensed individually, but OVRDDP's own code is not released for use under any license.

For clarity, I won't pursue any action against projects for personal use, but please do not use OVRDDP in projects intended for public release, particularly do not use OVRDDP in projects you intend to or are selling. Forks of OVRDDP are permitted even if released publicly.
